test/contract/audit/audit_test.go

```diff
@@ -86,12 +87,13 @@ func TestSyncs(t *testing.T) {
 			),
 		},
 	}
-	contract.RunContractTest(t, "test syncs", func(ct *contract.ContractTest) {
+	contract.RunGoContractTest(ctx, t, "test syncs", func(ch contract.ContractHelper) {
 		projectName := "audit-syncs-project"
-		ct.AddResources(time.Minute, contract.DefaultAtlasProject(projectName))
-		testProjectID := mustReadProjectID(t, ct.Ctx, ct.K8sClient, ct.Namespace(), projectName)
+		require.NoError(t, ch.AddResources(ctx, time.Minute, contract.DefaultAtlasProject(projectName)))
+		testProjectID, err := ch.ProjectID(ctx, projectName)
+		require.NoError(t, err)
 		ctx := context.Background()
```

Comment:
Not needed once there's already one on line 33

Reply:
Done